apps/gitness/src/components/RootWrapper.tsx

```diff
@@ -9,15 +10,28 @@ const RootWrapper = () => {
   const { currentUser } = useAppContext()
   const navigate = useNavigate()
   const { t } = useTranslation()
+
+  // Update type
+  const [recentRouteIDs, setRecentRouteIDs] = useState<any[]>([])
```

Comment:
Could you please add the export of types from the UI and use them here? In `canary/packages/ui/src/components/index.ts`, you need to add `export * from './navbar/types'`. Then, use the corresponding types for the states you are passing as props to the SandboxRoot component in your code.